src/main/java/com/epam/test_generator/services/utils/UtilsService.java

```diff
@@ -18,6 +20,28 @@
             throw new NotFoundException();
     }
 
+    public static void checkProjectIsActive(Project project) {
+        if (!project.isActive())
```

Comment:
Please use curly brackets in if-statements

Reply:
done